UPGRADING.md

```diff
@@ -1,4 +1,20 @@
-## Upgrading Mongoid Locker
+## Upgrading Mongoid-Locker
+
+## Upgrading to 2.0.0
+Mongoid-Locker supports only `5`, `6` and `7` versions of Mongoid.
+Since this version `Mongoid::Locker` uses unique name of locking and time is set by MongoDB. `Mongoid::Locker` no longer uses `locked_until` field and this field may be deleted. You must define new `locking_name` field of `String` type.
```

Comment:
Users should be told to call `Model.unset(:locked_until)` if they want to clean this up? Maybe add.